dropbear/localoptions.h

```diff
@@ -8,6 +8,10 @@
 #define ECDSA_PRIV_FILENAME "/local/dropbear_ecdsa_host_key"
 #define DROPBEAR_USE_PASSWORD_ENV 1
 
+
+/* enable unsafe DH group 1 to maintain compatibility with dropbear < 0.53 */
+#define DROPBEAR_DH_GROUP1_CLIENTONLY 0
```

Comment:
Could you add it through project DROPBEAR_CUSTOM_FLAGS (similar to ENABLE_PS_LOGIN_SERVICE).